.github/workflows/uploadbuildtomuuktest.yml

```diff
@@ -1,67 +1,130 @@
FOeLtWRuMBWg+XopNLQQitDg1d7DLzolMJFDdlOCvVQVU+NMzn1iWqIqGype0rtQmIkkv1GeWh+5/DTNnA7epvlbrcw5BTmlPA+ZFZxj04DSJZf0RUwHyr1Gx8/Njm06GfyrEwFL35/ysAdPB7PpWiFijlkIzdXO1gIk/k+/bsXWzByCtamAbBpB2IKzE4CGG4tIAwqE4DplwMet1o0ZZ4+5l4hZsHfwa9R7kjNAXl8v8rDTkELjjvMhZMatvthS0rRssjXtr71e/APZfqfP7sYYgQOZJsFf/TJ/Vt/mN4R+n6w3cpq8jli7A0Qhh84nvcm47WLDn2hbdWdOBivlH/OFfD/eMFI5IyuF+xzHbdFvub/KyE0L3g65eq0JQRzkN/cIgUmLP8WDzQRtPZx9yyrFiybwW5hwNOP2vcMbZCa/O02Taxp8Gbg3JACkCKx1vWnWeq0vYSFCjB9o/di4Rpbfmy4PoYItT+iPMbCVgr468FYoY0felbuJpI1VpzWIiW7yfZkhap9zDQ/M+M1qEMatXZHvLpNBWT2RRtFddKrgGSW0NIJRpvdyf/tuO91gMbHJNwuLbm8yHhdIu8ZVZp2KAuIP5gPOYVlHidRXBST0JDn82LAJ5UEbil/SRwIGlzwoEk2K32zG9v5BTlN2ATA2Dn/HfiNiItHs9rOfIqIFPE+hFaOGjGqh6TkKXsb/XhQXs2JbXHb0Xw7MeSNBI+6hsuxI6ASJlM4dKOBJZZJurPVuTi1PpGniQ2YutJ2NlnLWFUM9ZHAqoq2aLk7uQU6guXY22qSV+DWXNDo/GjctBpyMBU1H1D8sTqGxXFsbLu6tEUaXaMAdDfn1+duW9F76gyPFp+Gm27PC31sE9+ceb0dONHP3deWmBfIUPAQ7Eose694Mg3QSOlAU0PoFp8itXq8Ko5lD/khNiuzhCuPZZSNR+vyR+12rAHHImrWDWSdtDOMIIFvQYJKoZIhvcNAQcBoIIFrgSCBaowggWmMIIFogYLKoZIhvcNAQwKAQKgggTuMIIE6jAcBgoqhkiG9w0BDAEDMA4ECAZfjOK68VD4AgIIAASCBMhD52CVFjrv+1XIeCqluoO/xlOhnvh0YjafUoFAjZk5dJRAOpdpGSpQvKz4CIdobIiSNkneApux56PQRtvZtXAjmUMRS+wgCAsYUaky1pEzj3I9fTrn/xTKubk2rAwWI0jYwNPTJnF1ZErsCwIlDXiR0XKZy4YeFcTH9Hx7qbmNOvNUJ2U1IGwF2Zxni9vWDU8nqh+gO9Iu1Qy2ON+CL/apG4eGmCV1pgEWFGm/EKLsbr5b1hDMbY5YLNcq5Wj/X8Mrcefrr26Co8sZfsvDoFhX0vcYBLt+rfOZnN+xLVTtKYHNJxNvhwmqo7PiT8St8o+OSIQHFzowlfxcGytQddemE0pT86ecG/aP8O2fTf7+B/41jrsqVqk56IZ4BeKgs3/hAZezn4YgIt3ckeuFjXySCFzXDqZcwOsmfuydIV7dRl1RdMBnrv8424V00MfezRS40gSOMbaoLvLgQB37MmTdyz51KBNITLtuGpruyuNwdDC8iE7EO7/0p7VW+g800lBmUUbxbVpGecfiCZ5Y8rQSTjZkBRF2upvb5spYd7vSYIt7TkxN7NwVvGzVAd7pHashOevxS0eJy+Vg8NwQOCahHVHrCfu8faafma1bB98JyrgFNeY5MewNLWXKdpHMzJ4JLg8PflXiLxgkeLlj6gmiHQLZpsvLjPaXLy7wCdKTn8E/ZBzu5RJAceqX80jJsjpVaARObCQ/tABs/vZ0SgnMJ0xgo1uWcdIRCfVaTPr9uBNp4I2KWOkRIzPXBCAC9nFYkxZdlBaYaJbVB8kB0ORQuBJoBpZM/RxhRts+XbLjn1fzhR0Or2BQWOyoKBcWTd6dC8kRZPJBrn2zxMkJ0UNa7fB8Ji4JnotWwI2m6K7yLQeKcKg04yFmGWq4zv+bPtupjbCYrGY74FkQmld95w8tKY8P8huWWfZXNv+W88jDOrz9QBnbS/5qlM5yfNcEQo7+x77XKQAMcIoU5pmT3IYOhh1IUfV6XABYzjlwXMwrZ8rAdC5VyCiNKDz/rooirGYqy/lXmf55fzBMliB2FPJGg82Slu7lDCagPeVi0aNAN3F/5qhTPSAttXFcdH+sMfhQJF4ru96m+vQKrJjgWlmFPOo+BlaIMTB7E/B3+nhtfh8RiirSrYt6gzUsOKp0g0WAyH86gyOEDWZouzs5mpynitlfDt5h9Y6LaYcB+Hp3uLpq9ulm21RHaZ1i+j5pNnjaqYO5AGqzeUHOmPXYedBFyXhwm5k97QeSNjjTvJkAgtlkVgcUgyoKT0iM4DsDrwU0hEr9j44ozrkGzN7F4UBfzzO1EDHOUzlfo99e1I/QcRn65yu2swSGDAQF76YiOR31UppTVNzwHlEQFIkbv3VmjW2KzXqK/Y9XhwX4YLo/EMc1bmPrx+aBjL7jUFNpxE7l/xtbh/Z0JUZVHoz9wIuQ6WoVktRIskHKU2d4oqGiz4Gw9cMd+78leXguNbWcUUQh4J4Co7YBo1ixRvPCxqefyLCBte0rK3NcLA8n/tacOVx7X0MN0x8nDbcR4X+mZ4hRZTvLWI/ajNET8e0RvBQhT3zJar+5SfhIIsOm7X/klnL9+fJocVm10w+c7jZIREj31dYzWSOqtQbMiC84mChJfiMqvaBbobMxgaAweQYJKoZIhvcNAQkUMWweagBBAHAAcABsAGUAIABEAGUAdgBlAGwAbwBwAG0AZQBuAHQAOgAgAFAAcgBhAGsAYQBzAGgAIABLAG8AdAB3AGEAbAAgACgAUABvAHIAdABhAGwAIABMAGEAYgBzACwAIABJAG4AYwAuACkwIwYJKoZIhvcNAQkVMRYEFKHU0GxbPr7c4g5PlunT2GbxEMR4MDAwITAJBgUrDgMCGgUABBTnM2shzm/gGiZvvfcU8edl49MKeAQIon/g3GfSd4YCAQE=" 
```

Comment:
If these values aren't dynamic, we can probably move these to Github secrets.

Reply:
@blakeofwilliam 
We need to move all 4 items to secrets and all these items are not dynamic, so that I can use secrets.xxx for these values:
BUILD_CERTIFICATE_BASE64
BUILD_PROVISION_PROFILE_BASE64
EXPORT_OPTIONS_PLIST
MUUKTEST_API_KEY